### DISCUSSION TITLE
Support for the new models with the 11th processor

### DISCUSSION THREAD

**Discussion by leopseft** [AndresGarciaSobrado91/MSI-Modern15-Hackintosh#14] [Q&A] [answered]
Has anyone tried for the newer modern 15 models? Like the Modern 15 A11M.

CPU: 11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz
GPU: integrated Intel Corporation TigerLake-LP GT2 [Iris Xe Graphics]
Network: Intel Corporation Wi-Fi 6 AX201
NVME: Kingston
RAM: 16GB

**Comment by AndresGarciaSobrado91** [comment, marked as answer]
nope 😶
